variables.tf

```diff
@@ -43,4 +43,12 @@ variable "volume_path" {
 }
 variable "vpc_id" {
     type = string
+}
+
+variable "schedule" {
+    default = "cron(0 5 * * ? *)" 
+}
+
+variable "delete_after" {
```

Comment:
Rename to `backup_retention` and specify in the description that the unit is days.